Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
(optional): Reuse old code to change version instead of add and remove the whole block so that reviewers can see different easier.